baldrick/plugins/tests/test_github_pull_requests.py

```diff
@@ -21,7 +22,9 @@ def setup_module(module):
 
 
 def teardown_module(module):
-    PULL_REQUEST_CHECKS[:] = module.PULL_REQUEST_CHECKS_ORIG[:]
+    global PULL_REQUEST_CHECKS
```

Comment:
@adrn - you don't need global since you are modifying in-place. I'll just do the fix and merge.